packages/node_modules/@webex/plugin-meetings/src/media/index.js

```diff
@@ -400,6 +400,14 @@ Media.getMedia = (audio, video, config) => {
       false,
     fake: process.env.NODE_ENV === 'test' // Special case to get fake media for Firefox browser for testing
   };
+  const {sendAudio, sendVideo} = await Media.getSupportedDevice({sendAudio: mediaConfig.audio, sendVideo: mediaConfig.video});
```

Comment:
The parent function already does the check for the supported device , So you don't need this

Reply:
I tried removing getSupportedDevice function here but it didn't work. Also, it seem getSupportedDevice not exposed to webclient because i got undefined calling this function in webclient. @arun3528 I am not sure how this function would be called from parent function

Reply:
I exposed getSupportedDevice function so we can make call from web client now we don't need function here anymore.

Reply:
https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/@webex/plugin-meetings/src/meeting/index.js#L2675